vaadin-dialog-flow-parent/vaadin-dialog-flow-integration-tests/src/test/java/com/vaadin/flow/component/dialog/tests/DialogTestPageIT.java

```diff
@@ -162,6 +162,18 @@ public void openDialogAddComponentAtIndex() {
         assertButtonText(1);
     }
 
+    @Test
```

Comment:
Nit: Could group this test with the other tests that use this dialog instance and that are related to resizing / dragging:

Reply:
Into a separate file, you mean?

Reply:
No, there is a block of tests below related to the resizing / draggable dialog that is also used in this test.

Reply:
Got it. Done.